src/nimblepkg/nimscriptwrapper.nim

```diff
@@ -42,12 +43,12 @@ proc execNimscript(nimsFile, projectDir, actionName: string, options: Options,
 
   let
     cmd = ("nim e --hints:off --verbosity:0 -p:" & (getTempDir() / "nimblecache").quoteShell &
-      " " & nimsFileCopied.quoteShell & " " & actionName).strip()
+      " " & nimsFileCopied.quoteShell & " " & outFile.quoteShell & " " & actionName).strip()
+
+  displayDebug("Executing " & cmd)
 
   if live:
     result.exitCode = execCmd(cmd)
-    let
-      outFile = nimsFileCopied & ".out"
     if outFile.fileExists():
```

Comment:
Do we still need this? The outFile is now in /tmp so it should be removed by the OS.

Reply:
No harm deleting it either.